spec/prompts_spec.rb

```diff
@@ -153,9 +153,15 @@
         expect(subject.ask_for_ip('prompt', '1.1.1.1')).to eq('dead:beef::1')
       end
 
-      it "should handle hostname" do
-        say %w(5x redhat.com)
-        expect(subject.ask_for_ip_or_hostname("prompt", "1.1.1.1")).to eq("redhat.com")
+      it "should handle hostname beginning with a digit" do
+        say "5x"
```

Comment:
I would prefer we change this spec to something that actually looks more like an actual hostname. How about `198.51.100.1.example.com`